src/frontend/src/lib/components/wizards/migration/views/AlreadyMigrated.svelte

```diff
@@ -25,21 +26,28 @@
         Identity already upgraded
       </h1>
       <p
-        class="text-md text-text-tertiary font-medium text-balance sm:text-center"
+        class="text-md text-text-tertiary mb-2 font-medium text-balance sm:text-center"
       >
         {`${name} is already upgraded to the new experience.`}
       </p>
+      <!-- Removed "text-balance" because it breaks the button name `"Use existing Passkey"` into two lines -->
+      <p class="text-md text-text-tertiary font-medium sm:text-center">
+        Select "Use existing Passkey" in the login process.
```

Comment:
Let's highlight with styling instead, similar to how we're doing it in other views.

```suggestion
        Select <b class="text-text-primary">Use existing Passkey</b> in the login process.
```